ament_cmake_test/cmake/run_test.py

```diff
@@ -19,8 +19,8 @@
 import errno
 import os
 import re
-import sys
 import subprocess
+import sys
```

Comment:
I guess you had to run the test manually to find those? they are not tested automatically right?

Reply:
Those showed up in my awesome :atom: so I "had to fix" them while I was editing the file :rocket: